src/h_vialib/secure/token.py

```diff
@@ -47,9 +47,7 @@ def verify(self, token):
 
         try:
             return jwt.decode(token, self._secret, self.TOKEN_ALGORITHM)
-        except InvalidSignatureError as err:
-            raise InvalidToken("Invalid secure token") from err
         except ExpiredSignatureError as err:
             raise InvalidToken("Expired secure token") from err
-        except DecodeError as err:
-            raise InvalidToken("Malformed secure token") from err
+        except JWTError as err:
```

Comment:
python-jose doesn't expose these differently. In any case we are wrapping them in the same `InvalidToken` exception in our end.